pkg/cmd/infra/router/template.go

```diff
@@ -55,6 +58,7 @@ func (o *TemplateRouter) Bind(flag *pflag.FlagSet) {
 	flag.StringVar(&o.DefaultCertificate, "default-certificate", util.Env("DEFAULT_CERTIFICATE", ""), "A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format")
 	flag.StringVar(&o.TemplateFile, "template", util.Env("TEMPLATE_FILE", ""), "The path to the template file to use")
 	flag.StringVar(&o.ReloadScript, "reload", util.Env("RELOAD_SCRIPT", ""), "The path to the reload script to use")
+	flag.StringVar(&o.ReloadIntervalString, "interval", util.Env("RELOAD_INTERVAL", ""), "Controls how often router reloads are invoked. Mutiple router reload requests are coalesced for the duration of this interval since the last reload time.")
```

Comment:
Yeah, that's what it should be. Its more appropriate. I was going for 15s/1h initially but switched that around - so duration is a better option (no pun intended).